deps/build.jl

```diff
@@ -43,3 +44,46 @@ end
 compile(TZDATA_DIR, COMPILED_DIR)
 
 info("Successfully processed TimeZone data")
+
+
+if OS_NAME == :Windows #TODO change to window before commit
+    isdir(WINDOWS_DIR) || mkdir(WINDOWS_DIR)
+
+    # Windows is weird and uses its own timezone
+    info("Downloading Windows to TZ data")
+
+    # Generate the mapping between MS Windows timezone names and
+    # tzdata/Olsen timezone names, by retrieving a file.
+    xml_source = "http://unicode.org/cldr/data/common/supplemental/windowsZones.xml"
+    xml_file = joinpath(WINDOWS_DIR, "windowsZones.xml")
+    # Download the xml file from source
+    download(xml_source, xml_file)
+
+    # Get the timezone conversions from the file
+    xdoc = parse_file(xml_file)
+    xroot = root(xdoc)
+    windowsZones = find_element(xroot, "windowsZones")
+    mapTimezones = find_element(windowsZones, "mapTimezones")
+    # Every mapZone is a conversion
+    mapZones = get_elements_by_tagname(mapTimezones, "mapZone")
+
+    # Dictionary to store the windows to timezone conversions
+    win_tz = Dict{String,String}()
+
+    # Add conversions to the dictionary
+    for mapzone in mapZones
+        if attribute(mapzone, "territory") == "001"
```

Comment:
Why use territory "001"? Is there any documentation regarding this file you can link to?